nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/authorization/ProcessGroupAuthorizable.java

```diff
@@ -32,13 +33,28 @@ public interface ProcessGroupAuthorizable extends AuthorizableHolder {
      */
     ProcessGroup getProcessGroup();
 
+    /**
+     * Returns the Parameter Context Authorizable. May be null if the underlying Process Group is not
+     * bound to a Parameter Context.
+     *
+     * @return the Parameter Context authorizable
+     */
+    Authorizable getParameterContextAuthorizable();
+
     /**
      * The authorizables for all encapsulated processors. Non null
      *
      * @return all encapsulated processors
      */
     Set<ComponentAuthorizable> getEncapsulatedProcessors();
 
+    /**
+     * The authorizables for all encapsulated processors that meet the specified predicate. Non null
+     *
+     * @return all encapsulated processors
+     */
+    Set<ComponentAuthorizable> getEncapsulatedProcessors(Predicate<org.apache.nifi.authorization.resource.ComponentAuthorizable> filter);
```

Comment:
Since the method has a comment, it would be helpful to add a description for the `filter` parameter.